recipe_scrapers/rosannapansino.py

```diff
@@ -12,6 +12,10 @@ def host(cls):
     def title(self):
         return self.soup.find("meta", {"property": "og:title"})["content"]
 
+    def total_time(self):
```

Comment:
In case where site does not have time information for their recipes:

```
raise RecipeScrapersExceptions(f"{self.host} does not provide time information.")
```

or sth along these lines to set an example.